bms/bms_app/static/bms_app/css/create_project.css

```diff
@@ -4,9 +4,39 @@
 }
 
 .panel-body {
-    padding: 10px 10px 0px 10px;
+	padding: 10px 10px 0px 10px;
 }
 
+.page-header {
+	padding-bottom: 9px;
+	margin: 20px 0 20px;
```

Comment:
In shorthand properties 20 px(top) 0 (right) 20px(bottom) 0 (left); if top and bottom, right and left values are equal then no need to mention bottom and left. eg 20px 0; is sufficient.